### PR TITLE
Allow manually rebuild docker images

### PR DESCRIPTION
**Pull request by kezhenxu94**

### CODE REVIEW

File: .github/workflows/publish-docker.yaml [hunk 1]
```diff
@@ -20,6 +20,7 @@ on:
   push:
     branches:
       - main
+  workflow_dispatch:
```
Comment (wu-sheng) [inline .github/workflows/publish-docker.yaml:23]: Don't we need a version parameter to manually build?

Comment (kezhenxu94) [reply .github/workflows/publish-docker.yaml:23]: > Don't we need a version parameter to manually build?

No. We would either rebuild the latest master branch or need code changes in latest master. Rebuilding an old commit doesn't make sense.

Comment (wu-sheng) [reply .github/workflows/publish-docker.yaml:23]: I see. You want to rebuild the latest for local branch? Is that the purpose?

Comment (kezhenxu94) [reply .github/workflows/publish-docker.yaml:23]: yes

Comment (wu-sheng) [reply .github/workflows/publish-docker.yaml:23]: So, this would not work for fork repo, mostly it is for committers' branch.

**Review by wu-sheng — APPROVED** [review]

**Comment by kezhenxu94** [comment]
We should consider upgrade the Istio version in the tests, 1.7 ~ 1.9 looks really outdated

**Pull request merged**